api/audit-trail-gw/src/services/channel-info-service.ts

```diff
@@ -1,38 +1,66 @@
-import { ChannelInfo, ChannelInfoPersistence, ChannelInfoSearch, getDateFromString, getDateStringFromDate } from '@iota/is-shared-modules';
+import { ChannelInfo, ChannelInfoPersistence, ChannelInfoSearch, ChannelType, getDateFromString, getDateStringFromDate } from '@iota/is-shared-modules';
 import * as ChannelInfoDb from '../database/channel-info';
 import isEmpty from 'lodash/isEmpty';
 
 export class ChannelInfoService {
-	async getChannelInfo(channelAddress: string): Promise<ChannelInfo | null> {
+	async getChannelInfo(channelAddress: string, reqId: string, isAdmin: boolean): Promise<ChannelInfo | null> {
 		const channelInfoPersistence = await ChannelInfoDb.getChannelInfo(channelAddress);
-		return channelInfoPersistence && this.getChannelInfoObject(channelInfoPersistence);
+		const channelInfo = this.getChannelInfoObject(channelInfoPersistence);
+		const authorId = channelInfo?.authorId;
+		const hidden = channelInfo?.hidden;
+		const isInVisibilityList = channelInfo?.visibilityList.find(i => i.id == reqId) != null
```

Comment:
please add check just to be sure: channelInfo?.visibilityList && channelInfo.visibilityList.filter